data_collection/api/tracker_debugger_api.yaml

```diff
@@ -3346,10 +3346,6 @@ components:
           description: '[Custom variables](https://help.piwik.pro/analytics/custom-variables/) set on the ``page`` scope.'
           type: string
           example: '{"1":["AspectRatio","16:9"],"2":["Height","1080"],"3":["Width","1920"]}'
-        bots:
```

Comment:
There is `bots` parameter in the Collector Front, no in the Processor. We forget about it or removed intentionally?

Reply:
We had it in processor but it wasn't used.